src/frontend/static/frontend/src/components/pipeline/all-experiments-view/ClinicalSourcePopup.tsx

```diff
@@ -168,32 +171,37 @@ export class ClinicalSourcePopup extends React.Component<PopupClinicalSourceProp
-                    const jsonData = {
-                        headersColumnsNames,
-                        mRNASourceId,
-                        mRNASourceType: SourceType.UPLOADED_DATASETS,
-                        gemSourceId,
-                        gemSourceType: SourceType.UPLOADED_DATASETS
-                    }
+                getInputFileCSVFirstColumnAllRows(file, undefined).then((firstColumnAllRows) => {
+                    getInputFileCSVColumns(file, undefined).then((headersColumnsNames) => {
+                        // Sets the Request's Headers
+                        const myHeaders = getDjangoHeader()
+                        // Sends an array of headers to compare in server
+                        const jsonData = {
+                            clinicalData: firstColumnAllRows,
+                            mRNASourceId,
+                            mRNASourceType: SourceType.UPLOADED_DATASETS,
+                            gemSourceId,
+                            gemSourceType: SourceType.UPLOADED_DATASETS
+                        }
 
-                    ky.post(urlGetCommonSamplesClinicalSource, { json: jsonData, headers: myHeaders }).then((response) => {
-                        response.json().then((jsonResponse: DjangoNumberSamplesInCommonClinicalValidationResult) => {
-                            if (jsonResponse.status.code === DjangoResponseCode.SUCCESS) {
-                                if (jsonResponse.data.number_samples_in_common > 0) {
-                                    this.setState({ clinicalSource })
-                                } else {
-                                    this.clinicalSourceVoid()
+                        ky.post(urlGetCommonSamplesClinicalSource, { json: jsonData, headers: myHeaders }).then((response) => {
+                            response.json().then((jsonResponse: DjangoNumberSamplesInCommonClinicalValidationResult) => {
+                                if (jsonResponse.status.code === DjangoResponseCode.SUCCESS) {
+                                    if (jsonResponse.data.number_samples_in_common > 0) {
+                                        this.setState({
+                                            clinicalSource,
+                                            survivalTuplesPossiblesValues: headersColumnsNames
+                                        })
+                                    } else {
+                                        this.clinicalSourceVoid()
+                                    }
                                 }
-                            }
+                            }).catch((err) => {
+                                console.log('Error parsing JSON ->', err)
+                            })
                         }).catch((err) => {
-                            console.log('Error parsing JSON ->', err)
+                            console.log('Error getting user experiments', err)
```

Comment:
Ya que estamos, corregir el mensaje de error

Reply:
Dejar el comentario que editaste como estaba antes 'Error parsing JSON ->'. Ese estaba bien! Lo que está mal es el de abajo que dice "Error getting user experiments" cuando en realidad choca contra un endpoint que compara los Samples en común. Debería decir algo como "Error getting samples in common"